lib/embulk/output/google_spreadsheets.rb

```diff
@@ -134,16 +166,27 @@ def self.clean_previous_records(worksheet, schema, task)
         row_range    = task["row_index"]..worksheet.num_rows
         column_range = task["col_index"]...(task["col_index"] + schema.length)
 
-        row_range.each do |row|
-          column_range.each do |col|
-            worksheet[row, col] = ''
+        row_range.each do |r|
+          column_range.each do |c|
+            worksheet[r, c] = ''
           end
         end
       end
 
       def format(type, v)
-        return @null_representation if v.nil?
-        v
+        return @null_string if v.nil?
+
+        case type
+        when :timestamp
+          zone_offset = task['default_timezone']
+          format      = task['default_timezone_format']
+
+          v.dup.localtime(zone_offset).strftime(format)
```

Comment:
localtime("UTC") fails. We need a special care for "UTC", just skipping localtime()